### PR TITLE
refactor: use SvelteKit instead of plain Svelte

### LINKED ISSUE #8: feat: SvelteKit
### Description

Explore if SvelteKit should be used instead of pure Svelte.

### Motivation

The SvelteKit-native router is considered easier to handle than the currently used third-party library (`svelte-navigator`).
Also, an improvement in the developer experience is expected through using SvelteKit (such as relative paths, e.g. `$lib`).

### Resources

_No response_

### To-do List

- [ ] create a parallel branch, install new SvelteKit from scratch
- [ ] move components and logic to SvelteKit
- [ ] extensive testing, further exploration (mainly routing and rendering capabilites, async, fetch, etc.)

### PR DESCRIPTION
**Pull request by daniel-mader**
# Description of change
Please write a summary of your changes and why you made them.

## Links to any relevant issues
* #8 

## How the change has been tested
Describe the tests that you ran to verify your changes.
Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Definition of Done checklist
Add an `x` to the boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/hooks.server.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import type { Handle } from '@sveltejs/kit';
+
+export const handle = (async ({ event, resolve }) => {
+  // if (event.url.pathname.startsWith('/custom')) {
+  //     return new Response('custom response');
+  // }
+
+  //   console.log('hooks.server.ts: event:', event);
```
Comment (nanderstabel) [inline src/hooks.server.ts:8]: Can this be removed?

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <div
+      class="hide-scrollbar flex space-x-4 overflow-x-auto bg-gradient-to-r from-red-200 to-red-300 p-4 shadow-md"
+      in:fly={{ y: -64 }}
+      out:fly={{ y: -64 }}
+    >
+      <div class="flex-shrink-0 px-4 py-1 font-medium text-red-700">dev mode</div>
+      <button
+        class="rounded-full bg-red-300 px-4 py-1 text-sm font-medium text-red-700 hover:outline-none hover:ring-2 hover:ring-red-700 hover:ring-opacity-60"
+        >back</button
+      >
```
Comment (nanderstabel) [inline src/routes/+layout.svelte:36]: Without the svelte-navigator this is not functioning anymore. Is that deliberately? Should we remove it (for now)?

**Review by nanderstabel — APPROVED** [review]
Looks good :+1:

File: src/hooks.server.ts (hunk 1 above)
Comment (daniel-mader) [reply src/hooks.server.ts:8]: Yes, will remove it in the next PR (for convenience reasons ..)

**Pull request merged**